lib/matplotlib/figure.py

```diff
@@ -2890,8 +2890,12 @@ def savefig(self, fname, *, transparent=None, **kwargs):
 
         transparent : bool
             If *True*, the Axes patches will all be transparent; the
-            figure patch will also be transparent unless facecolor
+            Figure patch will also be transparent unless facecolor
             and/or edgecolor are specified via kwargs.
+
+            If *False* has no effect and leaves the color of the Axes and
+            Figure patches as they are.
```

Comment:
Can we clarify "...as they are"?  Maybe "as set by ``facecolor`` kwarg."?

Reply:
Added a note about that and ended up re-ordering the signature and the docstring to put what is in my judgement is more useful higher up.